Update async_and_await.md (#29743)
TheOdinProject/curriculum: javascript/asynchronous_javascript_and_apis/async_and_await.md

</script>
```

> Since `await` does not work on the global scope (Unless using a module script), we will have to create an `async` function that wraps our API call to Giphy.

Reviewer: ```suggestion
Since `await` does not work in the top level of a non-module script, we will have to create an `async` function that wraps our API call to Giphy.
```
@JustWaveThings opinion on the wording suggestion above? Avoids "global scope" because technically the top level of a module is *not* global. Personally, I feel the Safari top-level await limitation (just researched - really, Safari??) is a non-issue as far as TOP is concerned, and by the time users may end up genuinely facing that, I doubt they'd be incapable of navigating the problem themselves (I doubt that'd likely be in TOP anyway).
